docs/pages/develop/development-builds/introduction.mdx

```diff
@@ -1,88 +1,78 @@
+If instead you tried to use a library that is not included, e.g. [react-native-firebase](https://docs.expo.dev/guides/using-firebase/#using-react-native-firebase), then you can use the JS code and hot reload the new bundle into Expo Go but it will immediately error because the JS code is trying to call into the native code from the React Native Firebase package that does not exist in Expo Go. There is no way to get the native code into the Expo Go app unless it was already included in the bundle that was uploaded to the App / Play Store.
 
-</Collapsible>
+### Test changes in app icon, name, splash screen
 
-<Collapsible summary="How are development builds different from Expo Go?">
+If you're developing your app in Expo Go only, you _can_ build a store version of it that will use your provided values and images for there, it just won't be possible to test it in Expo Go.
 
-When you start a new project with Expo, the fastest and easiest way to get to "Hello, world!" is with [Expo Go](/get-started/set-up-your-environment/) on your device because you don't need to compile any native code or install any native tooling.
+All of these are native assets that get shipped with the native bundle and are immutable once installed. The Expo Go app does show a "splash screen which" which is just your app icon on a solid color background. This is a dev-only emulation of a splash screen we added so you could see what the splash screen will probably look like. It is limited however, e.g. you cannot test `SplashScreen.setOptions` to animate the splash screen.
 
-Beyond "Hello, world!" and prototypes, you'll quickly encounter limitations and need to build a development build of your app. For example, the Expo Go sandbox environment is limited to only the native packages included in the [Expo SDK](/versions/latest/), while you can include any library in a development build because it's just a normal native app.
+### Remote push notifications
 
-You can think of a development build as your fully customizable version of Expo Go, tailored specifically for your project. Or, conversely, Expo Go is a development environment with a preset runtime.
+While in-app [notifications](https://docs.expo.dev/versions/latest/sdk/notifications/) are available in Expo Go, remote push notifications (that is, sending a push notification from a server into the app) are not. This is because a push notification service should be tied to your own push notifications certificates and while it is possible to make it work in Expo Go, it often caused confusion for production builds. It is therefore better to test remote push notifications in Development Builds so you can ensure parity in behavior between development and production.
```

Comment:
```suggestion
While [in-app notifications](https://docs.expo.dev/versions/latest/sdk/notifications/) are available in Expo Go, remote push notifications (that is, sending a push notification from a server to the app) are not. This is because a push notification service should be tied to your own push notification certificates, and while it is possible to make it work in Expo Go, it often causes confusion for production builds. It is recommneded to test remote push notifications in development builds so you can ensure parity in behavior between development and production.
```

-"...while it is possible to make it work in Expo Go, it often caused confusion for production builds." I think you can omit this since SDK 53, push notifications will be completely unavailable.